react/week2/todo/src/components/Timer.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+
+class Timer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { count: 0 };
+  }
+
+  componentDidMount() {
+    this.timerId = setInterval(() => this.tick(), 1000);
+  }
+
+  componentWillUnmount() {
+    this.timerId = clearInterval();
+  }
+
```

Comment:
Good idea. Do you understand why to do this?

Reply:
Yes, we use clearInterval to cancel setInterval.